engine/admin/live-restore.md

```diff
@@ -5,7 +5,7 @@ title: Keep containers alive during daemon downtime
 ---
 
 By default, when the Docker daemon terminates, it shuts down running containers.
-Starting with Docker Engine 1.12, you can configure the daemon so that
+Starting with Docker Engine 1.12, you can configure the daemon on unix docker hosts so that
```

Comment:
With the "note", I'm ok with reverting this change, as it makes the sentence a bit harder to read